relay-server/src/services/processor/span/processing.rs

```diff
@@ -590,6 +590,31 @@ fn normalize(
 
     populate_ua_fields(span, user_agent.as_deref(), client_hints.as_deref());
 
+    // INP spans sets some top level span attributes inside span.data so make sure to pull
+    // them out to the top level before further processing.
+    if let Annotated(Some(ref mut data), _) = span.data {
+        if let Some(exclusive_time) = match data.other.get("sentry.exclusive_time") {
+            Some(Annotated(Some(Value::I64(exclusive_time)), _)) => Some(*exclusive_time as f64),
+            Some(Annotated(Some(Value::U64(exclusive_time)), _)) => Some(*exclusive_time as f64),
+            Some(Annotated(Some(Value::F64(exclusive_time)), _)) => Some(*exclusive_time),
+            _ => None,
+        } {
+            data.other.remove("sentry.exclusive_time");
+            span.exclusive_time = exclusive_time.into();
+        }
+
+        if let Some(profile_id) = match data.other.get("profile_id") {
+            Some(Annotated(Some(Value::String(profile_id)), _)) => profile_id
+                .parse()
+                .map(|profile_id| EventId(profile_id))
+                .ok(),
+            _ => None,
+        } {
+            data.other.remove("profile_id");
+            span.profile_id = profile_id.into();
+        }
+    }
```

Comment:
nit: Should we move this to a helper function (similar to `populate_ua_fields`)?